task_2.js

```diff
@@ -1,6 +1,12 @@
-/**
- * მოცემულია "string" -ების "array" -ი.
- * დაწერეთ კოდი, რომელიც ამ "array" -ში იპოვის ყველაზე მოკლე სიგრძის მქონე
- * "string" -ს და შემდგომ გამოიტანეთ ამ ელემენტის ზომა, მნიშვნელობა და რომელ ინდექსზეა ის
- * "console.log"-ის გამოყენებით
- */
\ No newline at end of file
+const words = ["car", "sauce", "vitamine" , "apartment"]
+
+let shortestWord = 0;

```

Comment:
ამ ცვლადის უფრო ზუსტი სახელი იქნებოდა:
```suggestion
let shortestWordIndex = 0;

```